### PR TITLE
Joltbuild fix

### PR DESCRIPTION
**Pull request by AverageNerdz**
**Brief Description of What This PR Does**

This PR does some stuff...

**Related Issues**

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [ ] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [x] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

File: src/native/physics/StepListener.cpp [hunk 1]
```diff
@@ -1,22 +1,17 @@
-// ------------------------------------ //
```
Comment (hhyyrylainen) [inline src/native/physics/StepListener.cpp:1]: Please keep my formatting style intact.

File: src/native/physics/StepListener.cpp [hunk 2]
```diff
@@ -1,22 +1,17 @@
...
 }
 
-void StepListener::OnStep(float inDeltaTime, JPH::PhysicsSystem& inPhysicsSystem)
+void StepListener::OnStep(const JPH::PhysicsStepListenerContext &inContext)
 {
-    // We assume here that the physics system is our target world's system
-    UNUSED(inPhysicsSystem);
-
-    notifyWorld.PerformPhysicsStepOperations(inDeltaTime);
+    // Assuming this operation uses the delta time from the context
```
Comment (hhyyrylainen) [inline src/native/physics/StepListener.cpp:13]:
```suggestion
    // Assuming this operation uses the delta time from the context and the target world is ours
```

File: src/native/physics/StepListener.cpp (hunk 1 above)
Comment (AverageNerdz) [reply src/native/physics/StepListener.cpp]: done pal, sorry about that i have a weird habit at changing that

**Review by hhyyrylainen — APPROVED** [review]
Fixes the build with Jolt, thanks.

**Pull request merged**